protocol-tests/src/main/java/fixtures/dpgcustomization/implementation/DpgClientImpl.java

```diff
@@ -709,6 +711,51 @@ public DpgClientImpl(HttpPipeline httpPipeline, SerializerAdapter serializerAdap
                                         null));
     }
 
+    /**
+     * Sends the {@code httpRequest}.
+     *
+     * @param httpRequest The HTTP request to send.
+     * @return the response body on successful completion of {@link Mono}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<BinaryData>> sendRequestAsync(HttpRequest httpRequest) {
+        return this.getHttpPipeline()
```

Comment:
How are we passing the context coming from Reactor? This should be using `withContext(context -> ...)`

Reply:
Ah, thanks for point it out. On the calling side for too long, and naturally think if I write `contextWrite` after `Mono` it will automatically get to the core Context. But that is because we had `FluxUtil.withContext` in generated code.